src/App.jsx

```diff
@@ -1,17 +1,35 @@
 import { hot } from 'react-hot-loader';
 import { Component } from 'react';
 import { BrowserRouter, Switch, Route } from 'react-router-dom';
+import { ApolloProvider } from 'react-apollo';
+import { ApolloClient } from 'apollo-client';
+import { HttpLink } from 'apollo-link-http';
+import { InMemoryCache } from 'apollo-cache-inmemory';
 import Projects from './views/Projects';
+import Project from './views/Project';
 
 @hot(module)
 export default class App extends Component {
+  cache = new InMemoryCache();
```

Comment:
I think we also want to persist the cache via https://github.com/taskcluster/taskcluster-web/blob/2998f6561a4de26908d4a2eba4aaca8443c15a53/src/App/index.jsx#L60-L63.